<launch>

   <!--Now apply segementation to remove table plane and generate clustered objects -->
  <node name ="super" pkg="sq_fitting" type="sq_fitter_node" output="screen">
      <param name="cloud_topic" value="/kinect2/sd/points" />
      <!--param name="cloud_topic" value="/camera/depth_registered/points" /-->


      <param name="zmax" value="2.0" />
      <param name="zmin" value="0.02" />
      <param name= "th_points" value="60" />

      <!--Supervoxel parameters -->
      <param name="voxel_resoluton" value="0.0075f" />
      <param name="seed_resolution" value="0.015f" />
      <param name= "color_importance" value="0.0f" />
      <param name="spatial_importance" value="1.0f" />
      <param name="normal_importance" value="4.0f" />

      <!-- LCCP parameters -->
      <param name="use_extended_convexity" value="false" />
      <param name="use_sanity_criterion" value="true" />
      <param name="concavity_tolerance_threshold" value="10" />
      <param name="smoothness_threshold" value="0.1f" />
      <param name="min_segment_size" value="3" />


      <rosparam param = "workspace"> [-0.23, 0.23, -0.35, 0.07, 0.95, 2.5]</rosparam>
      <!--rosparam param = "workspace"> [-0.5, 2, -0.2, 0.1, -2, 0.1]</rosparam-->

      <param name="remove_nan" value="true" />

  </node>



</launch>
